app/services/expired_letters.rb

```diff
@@ -0,0 +1,36 @@
+class ExpiredLetters
```

Comment:
Thoughts on renaming this something like `SendExpiredLetterNotifications`? At first glance the name of this makes me think it should be a model or something like that.